src/mem/ruby/system/VIPERCoalescer.cc

```diff
@@ -106,6 +107,10 @@ VIPERCoalescer::makeRequest(PacketPtr pkt)
         invTCP();
     }
 
+    if (pkt->req->isInvL2()) {
+        invTCC(pkt);
+    }
+
```

Comment:
probably should remove extra line here

Reply:
I was keeping the previous format of having an empty line before return statement (unless github is showing you something I am not seeing)